ta/pkcs11/src/persistent_token.c

```diff
@@ -538,6 +539,44 @@ void release_persistent_object_attributes(struct pkcs11_object *obj)
 	obj->attributes = NULL;
 }
 
+enum pkcs11_rc update_persistent_object_attributes(struct pkcs11_object *obj)
+{
+	TEE_Result res = TEE_ERROR_GENERIC;
+	TEE_ObjectHandle hdl = TEE_HANDLE_NULL;
+	uint32_t tee_obj_flags = TEE_DATA_FLAG_ACCESS_READ |
```

Comment:
Do we need read in here?

Reply:
You're right. For Write/Truncate, we don't need ACCESS_READ.

Reply:
Thanks for noticing it. I will remove both ACCESS_READ and TEE_DATA_FLAG_ACCESS_WRITE_META